tests/ci_install.sh

```diff
@@ -20,10 +20,9 @@ osx|macOS)
 esac
 
 # Disable excessive output
-mkdir -p $HOME/.config/yt
-echo "[yt]" > $HOME/.config/yt/yt.toml
-echo "suppress_stream_logging = true" >> $HOME/.config/yt/yt.toml
-cat $HOME/.config/yt/yt.toml
+echo "[yt]\nlog_level = 50" > yt.toml
```

Comment:
This is a change from before, yes?  Previously we turned off the stream, and now we're setting the level to 50.  This *may* be a distinction without a difference.

Reply:
yes it's a intentional change of behaviour. I don't think we have any errors logged in tests but if we do, I figure we might give ourselves a chance to see them